pod/video/templates/videos/video_stats_view.html

```diff
@@ -0,0 +1,94 @@
+{% extends 'base.html' %}
+{% load staticfiles i18n %}
+{% block opengraph %}
+    {{ block.super }}
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+{% endblock opengraph %}
+{% block page_title %}{% trans "Video - Statistics views" %}{% endblock page_title %}
+{% block page_extra_head %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.11.4/themes/redmond/jquery-ui.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/free-jqgrid/4.15.5/css/ui.jqgrid.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/free-jqgrid/4.15.5/jquery.jqgrid.min.js"></script>
+    <!-- https://free-jqgrid.github.io/getting-started/index.html -->
+    <script>
+	function linkedCell(cellValue, options, rowObject) 
+	{
+	    return "<a href='" + window.location.origin + "/video/"+
+	    rowObject.slug
+	    +"' target='_blank' rel='noopener'>"+cellValue+"</a>";
+	}
+	$(()=>{
+		let data_url = window.location.href;
+        	$("#grid").jqGrid({
+		    url: data_url,
+		    datatype: "json",
+		    mtype: "POST",
+        	    colNames: [
+                	gettext("Title"),
+	                gettext("View during the day"),
+        	        gettext("View during the month"),
+                	gettext("View during the year"),
+	                gettext("Total view from creation"),
+        	    ],
+		    colModel: [
+		    	{ 
+				name:"title",
+				align:"center",
+				sortable:true,
+				sorttype:"text",
+				formatter:linkedCell},
+		   	{ name: "day", align:"center", sortable:true, sorttype:"int"},
+	    		{ name: "month", align:"center", sortable:true, sorttype:"int" },
+	  	 	{ name: "year", align:"center", sortable:true, sorttype:"int" },
+		        { name: "since_created", align:"center", sortable:true, sorttype:"int" },
+		    ],
+		loadonce: true,
+		rowNum: 10,
+		rowList: [10, 15, 20, 25, 30, 50, 100],
+		gridview: true,
+		autoencode: true,
+	    	pager: '#pager',
+		sortorder: 'asc',
+		beforeProcessing: function(data){
+			// Set min date
+			let min_date = data.filter(obj => {
+				return obj.min_date != undefined;
+			})
+			// remove date_min in data
+			data.pop();
+			document.querySelector("#jsperiode").min = min_date[0].min_date;
+		},
+		postData: {
+			csrfmiddlewaretoken: $("[name=csrfmiddlewaretoken]").val()
+		}
+        });
+	let today = new Date().toISOString().split("T")[0];
```

Comment:
Please test it in multiple browser (google, firefox, safari and IE) and tell me if it's ok

Reply:
I'm not sure that "let" declaration is ok in all browser

Reply:
Only Opera and QQ Browser don't support "let" => https://caniuse.com/#search=let

Reply:
ok, thanks. Have you tried your JS code in several browser ?